### ISSUE TITLE
Gene glyph colors on Firefox

### ISSUE THREAD

**Issue by nvictus** [higlass/higlass#811]
Gene annotation glyphs (but not the gene label text) have weird colors on firefox:

![image](https://user-images.githubusercontent.com/1270651/70867506-2cea2100-1f44-11ea-8bdd-740fedfdde25.png)

The track label on the vertical track also appears black:

![image](https://user-images.githubusercontent.com/1270651/70867541-7b97bb00-1f44-11ea-962f-6ed6dee94901.png)

#### Steps to reproduce
1. open a gene annotation track in firefox

#### Observed behavior
- red glyphs are rendered cyan
- blue glyphs are rendered yellow
- transluscent white backgrounds rendered black

Seems like firefox is somehow applying a CMYK colorspace instead of RGBA. 

Further evidence: setting `minusStrandColor` to `rgba(0, 255, 0, 1)` makes the gene glyphs turn magenta, but the gene labels correctly turn green.

It doesn't seem to affect other tracks...

**Comment by flekschas** [comment]
This seems to be a very strange bug. I don't have this issue on FF `v71.0` with macOS `v10.14.6`

This is how the default viewconf looks like for me in FF

![Screen Shot 2019-12-15 at 3 15 22 PM](https://user-images.githubusercontent.com/932103/70868365-34162c80-1f4e-11ea-8096-22341b3d6b1a.png)

**Comment by flekschas** [comment]
I created a simple test case: https://jsfiddle.net/13h78dkq/
Strangely, it works for me and Nezar.

**Comment by flekschas** [comment]
Also interesting, after scaling up Nezar's screenshot one can see a couple of pixels with the correct color:

![Screen Shot 2019-12-15 at 4 58 04 PM](https://user-images.githubusercontent.com/932103/70872741-2b3a5080-1f78-11ea-901c-ae360f6047fd.png)

**Comment by nvictus** [comment]
Simplest example: http://higlass.io/l/?d=YwFgTeDHRrWWO6_Syq-n0w

![image](https://user-images.githubusercontent.com/1270651/70873340-65592180-1f7b-11ea-8106-d23931c34640.png)

**Comment by alexpreynolds** [comment]
Is this specific to a version of Firefox?

<img width="1035" alt="Screen Shot 2019-12-15 at 5 56 04 PM" src="https://user-images.githubusercontent.com/33584/70873827-47cc8d80-1f64-11ea-91ca-95380ab245b8.png">

**Comment by nvictus** [comment]
definitely affects 
![image](https://user-images.githubusercontent.com/1270651/70874130-c6362900-1f7e-11ea-9110-e6be1a53b382.png)

Also the thumbnails generated by pyppeteer seem to have this issue, and they are generated by a headless chromium!

**Comment by alexpreynolds** [comment]
If it affects multiple apps under Linux, perhaps it would be due to some shared graphics library (libpng etc.) that Firefox and Chromium rely upon when rendering?

**Comment by nvictus** [comment]
Indeed, I can't reproduce this now on Windows or Mac (though I could swear I had seen it on one of them before). So it could be linux-specific, though the colors render correctly for me on chromium on ubuntu.

**Comment by nvictus** [comment]
Tracked it down on Firefox to a GL feature disabled by default on X11 systems: https://wiki.mozilla.org/Blocklisting/Blocked_Graphics_Drivers#On_X11

```
GL layers acceleration is not yet enabled by default (see bug 594876). You can enable it by setting layers.acceleration.force-enabled=true in about:config.
```

Enabling `layers.acceleration.force-enabled` fixed the color issue. There must be some similar GL acceleration option disabled in the headless chromium that pyppeteer uses.

**Comment by flekschas** [comment]
I think there isn't anything we can fix on our side so shall we close this? We could add a note to the troubleshooting section (if we had one)?

**Comment by nvictus** [comment]
FYI, the actual feature that in question is [GPU-accelerated layer compositing](https://cialu.net/how-to-turn-on-accelerated-compositing-layers-on-firefox/). Solution: Enter `about:config` in Firefox's URL bar and set `layers.acceleration.force-enabled` to `true`.

However, when I tried it, it seemed to make HiGlass more unstable on Linux. There's probably a reason it is disabled. It probably depends on GPU drivers: https://www.reddit.com/r/firefox/comments/8hmnn7/firefox_gpu_acceleration_on_linux/
